x-pack/plugins/stack_connectors/common/crowdstrike/constants.ts

```diff
@@ -14,4 +14,7 @@ export enum SUB_ACTION {
   HOST_ACTIONS = 'hostActions',
   GET_AGENT_ONLINE_STATUS = 'getAgentOnlineStatus',
   EXECUTE_RTR_COMMAND = 'executeRTRCommand',
+  EXECUTE_ACTIVE_RESPONDER_RTR = 'batchActiveResponderExecuteRTR',
+  EXECUTE_ADMIN_RTR = 'batchAdminExecuteRTR',
+  GET_SCRIPTS = 'getScripts',
```

Comment:
Are these general scripts, or are they specific to RTR? If they’re specific to RTR, consider including ‘RTR’ in the name for clarity.

Reply:
They're related to RTR, added the prefix - thanks 👍